### ISSUE TITLE
Removing last comma in csv files

### ISSUE THREAD

**Issue by akrasuski** [firemodels/cfast#653]
Each row in csv file is now ended with comma. Is it possible to remove this last comma. While parsing automatically the csv file the problem manifests with empty last cell.

**Comment by rmcdermo** [comment]
Just use something other than Excel to parse the file.

**Comment by akrasuski** [comment]
I use Python csv for parsing a csv file. This is not the problem for me. However, it requires some extra-code. I've just wanted to point out, that the csv files don't converge with csv convention: [https://en.wikipedia.org/wiki/Comma-separated_values](url)

**Comment by rmcdermo** [comment]
Here is Python code I use the parse CFAST output.  I did not need extra code.
```
#!/usr/bin/python
#McDermott
#2016-02-01 20:52:39

from __future__ import division # make floating point division default as in Matlab, e.g., 1/2=0.5
import math
import numpy as np
import scipy.special as sp
import matplotlib.pyplot as plt
from matplotlib import rc
rc('font',**{'family':'sans-serif','sans-serif':['Helvetica'],'size':16})

#FirstTry
chid = 'FirstTry'

# read data from _n.csv file
M = np.genfromtxt(chid+'_n.csv', delimiter=',', names=True)
t = M['Time'][1:]
HRR = M['HRR_1'][1:]*1.e-3
MLR = M['PYROL_1'][1:]
LH = M['HGT_1'][1:]
TUpper = M['ULT_1'][1:]
```

**Comment by akrasuski** [comment]
I'm sorry Randy I wasn't probably clear enough. I don't need python code for extraction of a specific value from a specific CFAST csv output file. I'm doing something slightly different. 
I only wanted issue a problem that CFAST output files need some corrections. They are not fully compatible with the csv specification.

**Comment by rmcdermo** [comment]
I will let Rick decide if he wants to reopen the issue.

**Comment by rpeacoc** [comment]
I think this is an easy fix ... I'll check it out.

**Comment by akrasuski** [comment]
Thank you very much for help!

**Comment by rpeacoc** [comment]
This should be resolved in 8487a15 .
